cmd-runner/cmd_runner.go

```diff
@@ -0,0 +1,21 @@
+package cmd_runner
+
+import (
```

Comment:
Perhaps cmd_runner should be under pkg directory since you are already isolating this logic.

Reply:
Yeah, I think that would be better -- moving.